cpp/src/io/utilities/config_utils.cpp

```diff
@@ -52,11 +52,14 @@ bool is_gds_enabled() { return is_always_enabled() or get_env_policy() == usage_
 
 bool is_kvikio_enabled() { return get_env_policy() == usage_policy::KVIKIO; }
 
-void set_thread_pool_nthreads_from_env()
+void set_up_kvikio()
 {
   static std::once_flag flag{};
   std::call_once(flag, [] {
-    auto nthreads = getenv_or<unsigned int>("KVIKIO_NTHREADS", 8U);
+    auto compat_mode = kvikio::detail::getenv_or<bool>("KVIKIO_COMPAT_MODE", true);
```

Comment:
`kvikio::detail::getenv_or<bool>` is used here to be consistent with the way KvikIO captures the env var, which permits case-insensitive and wider range of options for boolean type: `true/yes/on`, `false/no/off`